File: fix.php
<?php
$file = 'resources/views/dosen/cetak_profil.blade.php';
$content = file_get_contents($file);
$content = str_replace('tahun_akademik', 'tahun_sekarang', $content);
file_put_contents($file, $content);
echo "Replaced all tahun_akademik to tahun_sekarang\n";

File: fix.py
content = open('resources/views/layouts/app.blade.php', 'r', encoding='utf-8').read()

part1_search = """                </li>
                @if(Auth::user()->hasPermission("digital-twin"))
                <li class="nav-item mt-2">"""
part1_repl = """                </li>
                @endif
                @if(Auth::user()->hasPermission("digital-twin"))
                <li class="nav-item mt-2">"""

part2_search = """                    </a>
                </li>
                @endif
                @endif
            </ul>"""
part2_repl = """                    </a>
                </li>
                @endif
            </ul>"""

content = content.replace(part1_search, part1_repl).replace(part2_search, part2_repl)

open('resources/views/layouts/app.blade.php', 'w', encoding='utf-8').write(content)
